addons/web_editor/static/src/xml/editor.xml

```diff
@@ -9,7 +9,7 @@
         <div id="toolbar" class="oe-toolbar oe-floating">
             <div id="style" class="btn-group dropdown">
                 <button type="button" class="btn dropdown-toggle"
-                    data-bs-toggle="dropdown" data-bs-original-title="Heading" tabindex="-1" aria-expanded="false">
+                    data-bs-toggle="dropdown" data-bs-original-title="Text style" tabindex="-1" aria-expanded="false">
```

Comment:
You introduced this in your first commit and are changing again here. I suggest you directly name it "Text style" rather than "Heading" in your first commit.

![image](https://user-images.githubusercontent.com/24205914/220920607-5d856df4-3570-4779-be70-ea1baee90f89.png)